docs/refs/configuring_mbed_os.md

```diff
@@ -19,27 +23,31 @@ The default stack size of main application thread is 4 kilobytes. This memory is
 ```
 
 #### Configuring user spawned thread stacks
-The default stack size of a user spawned thread is 4 kilobytes. The stack size of user spawned threads can be configured using the [mbed config system](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/config_system/) or passed in the [Thread constructor](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/group__rtos.html#gab9f45e13f619a725b87c1e43c14cf158). Reducing the default stack size for user spawned threads from 4K to 2K can be done in the following two ways:
-1. create an `mbed_app.json` file for your project with the following content:
-```JSON
-{
-    "config": {
-        "thread-stack-size": {
-            "value": 2000
+
+The default stack size of a user spawned thread is 4 kilobytes. You can configure the stack size of user spawned threads by using the [Mbed config system](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/config_system/) or passing in the [Thread constructor](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/group__rtos.html#gab9f45e13f619a725b87c1e43c14cf158). Reducing the default stack size for user spawned threads from 4K to 2K can be done in the following two ways:
+
+1. Create an `mbed_app.json` file for your project with the following content:
+
+    ```JSON
+    {
+        "config": {
+            "thread-stack-size": {
+                "value": 2000
+            }
         }
     }
-}
-```
+    ```
+
 2. Construct the thread like this in your application:
-`Thread thread(osPriorityNormal, 2000);`
 
-By default, the memory for the thread's stack is dynamically allocated from the global heap, though it can be changed in the [Thread constructor](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/group__rtos.html#gab9f45e13f619a725b87c1e43c14cf158). Here is an example application that uses memory allocated from the main thread's stack for the newly spawned thread stack:
+    `Thread thread(osPriorityNormal, 2000);`
+
+By default, the memory for the thread's stack is dynamically allocated from the global heap, though you can change it in the [Thread constructor](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/group__rtos.html#gab9f45e13f619a725b87c1e43c14cf158). Here is an example application that uses memory allocated from the main thread's stack for the newly spawned thread stack:
```

Comment:
Query: Would it be correct to change the first sentence to "By default, the global heap dynamically allocates the memory for the thread's stack, though you ..." for active voice?